infrastructure/helm/keycloak/login-setup-2.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+RELEASENAME=$1
+CLIENTSECRET=$2
+LOCALIP=$(ifconfig | grep 'inet ' | sed '/127/d' | cut -f 2 -d ' ')
+KEYCLOAKPORT=$(kubectl get svc $RELEASENAME-keycloak -o=jsonpath='{.spec.ports[0].nodePort}')
+
+# add 4180 port to $RELEASENAME-kuberay-head-svc
+kubectl patch svc $RELEASENAME-kuberay-head-svc  -p '{"spec":{"ports": [{"port": 4180,"name":"proxy","appProtocol":"tcp","Protocol":"TCP","targetport": 4180}]}}'
```

Comment:
Can this be in the template .yaml file?

Reply:
This service should be create in the kuberay helm.  I'll look into it.

Reply:
It is created by the kuberay operator based on the container ports of the ray head pod.  It overwrites default ports so all the ports must be specified.

```
ray-cluster:
  head:
    ports: [{containerPort: 10001, name: client},{containerPort: 6379, name: redis},{containerPort: 8265, name: dashboard},{containerPort: 8080, name: metrics},{containerPort: 8000, name: serve},{containerPort: 4180, name: proxy}]

```